client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -44,6 +44,9 @@ import { ModalViews } from 'state/ui/media-modal/constants';
 var REGEXP_IMG = /<img\s[^>]*\/?>/ig,
 	SIZE_ORDER = [ 'thumbnail', 'medium', 'large', 'full' ];
 
+let lastDirtyImage = null,
+	numOfImagesToUpdate = Number.MAX_SAFE_INTEGER;
```

Comment:
There's no support in IE for MAX_SAFE_INTEGER. Let's set this to something explicit like `null` and check for truthy or 0 before subtracting.